src/test/java/org/broadinstitute/consent/http/db/DataAccessRequestDAOTest.java

```diff
@@ -2,11 +2,7 @@
 
 import org.apache.commons.lang3.RandomStringUtils;
 import org.apache.commons.lang3.RandomUtils;
-import org.broadinstitute.consent.http.models.DarCollection;
-import org.broadinstitute.consent.http.models.DataAccessRequest;
-import org.broadinstitute.consent.http.models.DataAccessRequestData;
-import org.broadinstitute.consent.http.models.Dataset;
-import org.broadinstitute.consent.http.models.User;
+import org.broadinstitute.consent.http.models.*;
```

Comment:
Avoid * imports. There is a setting in intellij that lets you configure the value where it starts doing this for you. I set mine to 9999 or something crazy.

![Screen Shot 2022-06-16 at 11 10 32 AM](https://user-images.githubusercontent.com/116679/174101931-a2de4def-deee-4aaa-94f1-d94d01f0e4a3.png)